### PR TITLE
fix bug on dependencies validation function

### PR DESCRIPTION
**Pull request by denis99999**
Signed-off-by: Denis Barbaron <denis.barbaron@orange.com>

In some `X` modules the `package.js` file is used instead of the `package.json` file which leads to an `ERROR: X dependency is not installed`

### CODE REVIEW

**Comment by denis99999** [comment]
@koral--  don't merge this PR for now, I have also to update documentation, package.json (name, author, etc.), semaphore folder, .., I will do that when I will be back fron vacation

**Review by koral-- — APPROVED** [review]

**Comment by koral--** [comment]
Is that all the changes?

**Comment by denis99999** [comment]
Yes, I will complete the documentation later to update the link to npmjs once the package will be published!
I suppose the publishing will be done once the PR will be merged ?

**Comment by denis99999** [comment]
Are you merging or is it me?

**Comment by koral--** [comment]
> Yes, I will complete the documentation later to update the link to npmjs once the package will be published! I suppose the publishing will be done once the PR will be merged ?

Yeah, I've enabled this project on the semaphore CI console. So after merging this PR, everything defined here will start working.

**Pull request merged**
